MapboxDirections/MBDirectionsOptions.swift

```diff
@@ -328,7 +328,7 @@ open class DirectionsOptions: NSObject, NSSecureCoding, NSCopying {
      An array of directions query strings to include in the request URL.
      */
     internal var queries: [String] {
-        return waypoints.compactMap { return "\($0.coordinate.longitude.rounded(to: 1e6)),\($0.coordinate.latitude.rounded(to: 1e6))" }
+        return waypoints.compactMap{ $0.coordinate.stringForRequestURL }
```

Comment:
Note the following edge case: if the developer specifies three waypoints and one of them is invalid, this library previously sent a request with all three waypoints, all but guaranteeing an error in the response. Now it sends only two of the waypoints, which may return a valid response (ignoring the invalid coordinate). However, if the invalid waypoint had the `name` or `targetCoordinate` property set, then there would be a mismatch between the path and query parameters in the request URL, resulting in a different error than before in the response.